src/repr/src/relation.rs

```diff
@@ -339,6 +339,13 @@ impl RustType<ProtoColumnName> for ColumnName {
     }
 }
 
+impl From<ColumnName> for mz_sql_parser::ast::Ident {
+    fn from(value: ColumnName) -> Self {
+        // Note: ColumnNames are known to be less than the max length of an Ident (I think?).
```

Comment:
Looking through the code base, I see 80 instances of initializing a `ColumnName` I have no idea if we check in all those places that the name is less than the max allowed characters. We probably want to do something similar to `ColumnName` where we add a constructor that checks the length. We should probably add a followup GH issue.